package.json

```diff
@@ -42,7 +42,7 @@
     "event-loop-spinner": "^2.0.0",
     "gunzip-maybe": "^1.4.2",
     "mkdirp": "^1.0.4",
-    "packageurl-js": "^1.1.1",
+    "packageurl-js": "1.2.0",
```

Comment:
you also need to update `package-lock.json`

can be done by running `npm install` (and then run `npm ci` just to make sure the lock file is in sync with the package file)

Reply:
I did. It didn't change

Reply:
Since this is a library, I don't think package-lock.json is relevant.

Reply:
either way, it didn't affect packageurl-js (and it has no dependencies)

Reply:
do we want to allow clients of this lib to use an older version? (I don't think we need to, but if we do, then your original fix was probably better)

Reply:
we still have a lock file, so you could update it - that will make sure that we run local tests with the desired version

Reply:
I don't think we care all that much about which version they use.

I'll push the lockfile anwyay